docs/extensibility/theme-support.md

```diff
@@ -204,25 +204,6 @@ Enabling editor styles support is done using:
 add_theme_support( 'editor-styles' );
```

Comment:
I'd suggest rewriting this whole chunk, 194–204, with the following:

> Gutenberg supports the theme's [editor styles](https://codex.wordpress.org/Editor_Style), however it works a little differently than in the classic editor.
> 
> In the classic editor, the editor stylesheet is loaded directly into the iframe of the WYSIWYG editor, with no changes. Gutenberg, however, doesn't use iframes. To make sure your styles are applied only to the content of the editor, we automatically transform your editor styles by selectively rewriting or adjusting certain CSS selectors.
> 
> Because it works a little differently, you need to opt-in to this by adding an extra snippet to your theme, _in addition to_ the `add_editor_style` function:
> 
> `add_theme_support('editor-styles');`
>
> You shouldn't need to change your editor styles too much; most themes can add the snippet above and get similar results in the classic editor and inside Gutenberg.